contrib/pgmq/Trunk.toml

```
@@ -0,0 +1,18 @@
+[extension]
+name = "pgmq"
+version = "0.10.0"
```

```suggestion
version = "0.10.1"
```
This was recently bumped to `0.10.1` https://github.com/tembo-io/pgmq/blob/78756df25fbd05da7c70fcc034c267492a4f1bbe/Cargo.toml#L3